_sass/blocks/_revenues_steps.scss

```diff
@@ -5,14 +5,37 @@
 // Styleguide blocks.revenues_steps
 
 .revenues_steps-heading {
-	@include heading(h1);
+
+  @include respond-to(tiny-down) {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets